CurrencyConverterControl/Themes/Generic.xaml

```diff
@@ -0,0 +1,67 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="clr-namespace:CurrencyConverterControl">
+   <Style TargetType="{x:Type local:CurrencyConverterControl}">
+        <Setter Property="Background" Value="{x:Static SystemColors.WindowBrush}" />
+        <Setter Property="Foreground" Value="{x:Static SystemColors.ControlTextBrush}" />
+        <Setter Property="FontFamily" Value="{x:Static SystemFonts.CaptionFontFamily}"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type local:CurrencyConverterControl}">
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="*"/>
+                        </Grid.ColumnDefinitions>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="22"/>
+                            <RowDefinition Height="auto"/>
+                        </Grid.RowDefinitions>
+
+                        <!--Input Value to convert-->
+                        <TextBox  x:Name="TxtInput" Grid.Column="0" Text="{Binding InputValue, RelativeSource={RelativeSource AncestorType={x:Type local:CurrencyConverterControl}},
+                            UpdateSourceTrigger=PropertyChanged , StringFormat=N2}" 
+                            HorizontalAlignment="Left" Width="130" Height="22" Background="{TemplateBinding Background}" Foreground="{TemplateBinding Foreground}" FontFamily="{TemplateBinding FontFamily}"/>
+                        <ComboBox x:Name="CmbSource"  SelectedIndex="0"
+                            ItemsSource="{Binding RelativeSource={RelativeSource AncestorType={x:Type local:CurrencyConverterControl}},
+                            Path=CurrencyList}"
+                                  SelectedItem="{Binding SourceCurrency, RelativeSource={RelativeSource 
+                                  AncestorType={x:Type local:CurrencyConverterControl}}, Mode=TwoWay}"
+                                  Grid.Column="1" Height="22" Width="120" Background="{TemplateBinding Background}" 
```

Comment:
usually we try to have as simple and clean xaml as possible. It reduces time for xaml loading and layout. In most cases we apply Border settings and Background on some root panel in the ControlTemplate. In internal parts like ComboBox, it should be ok to set Background to Transparent